packages/front-end/components/Experiment/TabbedPage/SetupTabOverview.tsx

```diff
@@ -37,8 +40,13 @@ export default function SetupTabOverview({
   setChecklistItemsRemaining,
   envs,
 }: Props) {
-  const { apiCall } = useAuth();
   const [showHypothesisModal, setShowHypothesisModal] = useState(false);
+  const [showDescriptionModal, setShowDescriptionModal] = useState(false);
+
+  const expandDescription =
+    localStorage.getItem(`collapse-${experiment.id}-description`) === "true"
```

Comment:
Is there a reason to do this instead of `useLocalStorage`?

Reply:
I simply forgot about this hook. I'll refactor to use it for consistency.